### PR TITLE
Design Review - Loader & Dropdowns - [RET-417] [RET-420]

### PR DESCRIPTION
**Pull request by mhfen**
### What's this PR do?
- Always plays a loader out before killing the tween.
- Sets dropdowns in the body content to a set width
- Adds modal transitions

### Does this add new dependencies? Is there an installation procedure? (bundle, bower, npm, etc)
No.

### What are the relevant tickets?
https://mondorobot.atlassian.net/browse/RET-417
https://mondorobot.atlassian.net/browse/RET-420

### Who should review this PR?
@cprater @mhfen

### CODE REVIEW

**Comment by mhfen** [comment]
@cprater

**Comment by mhfen** [comment]
@cprater Rebased

File: src/assets/styles/modules/objects/dialog/_dialog.scss [hunk 1]
```diff
@@ -40,7 +42,7 @@
   position: absolute;
   right: 2.4rem;
   top: 2.4rem;
-  z-index: 25;
+  z-index: 1;
```
Comment (cprater) [inline src/assets/styles/modules/objects/dialog/_dialog.scss]: Does this still show on mobile Safari? I had to change the z-index previously

Comment (mhfen) [inline src/assets/styles/modules/objects/dialog/_dialog.scss]: I think this was a rebase reconciliation. I'll test and bump it if it needs to go up.

Comment (mhfen) [inline src/assets/styles/modules/objects/dialog/_dialog.scss]: value of 1 looks good on simulator.

**Pull request merged**